web/webkit/src/main/scala/net/liftweb/http/js/JsCommands.scala

```diff
@@ -646,7 +651,13 @@ trait HtmlFixer {
 }
 
 trait JsCmd extends HtmlFixer with ToJsCmd {
-  def &(other: JsCmd): JsCmd = JsCmds.CmdPair(this, other)
+  def &(other: JsCmd): JsCmd = {
+    if (other == JsCmds.Noop) {
+      this
+    } else {
+      JsCmds.CmdPair(this, other)
+    }
+  }
```

Comment:
I bet this means that [this change](https://github.com/lift/framework/commit/94bfcbfacafe2e5b7d017f4e5d8153f72499f5b1) is no longer needed :)